client/my-sites/themes/themes-banner/README.md

```diff
@@ -5,32 +5,47 @@ This component is used to implement a banner with image, text, and button.
 
 #### How to use:
 
+Add the following code into the same directory of this README file. Each banner should be on its own file.
+
 ```jsx
-import { ThemesBanner } from './themes-banner';
-
-const SampleBanner = () => (
-	<ThemesBanner
-		title={ translate( 'Are you a small business?' ) }
-		description={ translate(
-			"We understand your time contraints, and that's why we created this easy-to-set-up Premium theme called Small Business."
-		) }
-		action={ this.buttonAction }
-		actionLabel={ translate( 'Learn about this theme' ) }
-		backgroundImage="/calypso/images/example/background.jpg"
-		href="https://wordpress.com"
-	/>
-);
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+import ThemesBanner from './index';
+
+class SmallBusinessBanner extends PureComponent {
+	render() {
+		const { translate } = this.props;
+		return (
+			<ThemesBanner
+				themeId={ 'small-business' }
+				themeName={ 'Small Business' }
+				title={ translate( 'Do you run a small business?' ) }
+				description={ 'An awesome description for this theme' }
```

Comment:
Maybe wrap in a `translate` call? (Not too relevant, since it's just a README example, but it might be helpful for anyone who copy-pastes this.)

Reply:
True, good catch! Will update accordingly.